databallpy/match.py

```diff
@@ -218,6 +374,38 @@ def get_match(
             on="id",
         )
 
+    # add period column to tracking_data
+    period_conditions = [
+        (tracking_data["timestamp"] <= merged_periods.loc[0, "end_frame"]),
+        (tracking_data["timestamp"] > merged_periods.loc[0, "end_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[1, "start_frame"]),
+        (tracking_data["timestamp"] >= merged_periods.loc[1, "start_frame"])
+        & (tracking_data["timestamp"] <= merged_periods.loc[1, "end_frame"]),
+        (tracking_data["timestamp"] > merged_periods.loc[1, "end_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[2, "start_frame"]),
+        (tracking_data["timestamp"] >= merged_periods.loc[2, "start_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[2, "end_frame"]),
+        (tracking_data["timestamp"] > merged_periods.loc[2, "end_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[3, "start_frame"]),
+        (tracking_data["timestamp"] >= merged_periods.loc[3, "start_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[3, "end_frame"]),
+        (tracking_data["timestamp"] > merged_periods.loc[3, "end_frame"])
+        & (tracking_data["timestamp"] < merged_periods.loc[4, "start_frame"]),
+        (tracking_data["timestamp"] > merged_periods.loc[4, "start_frame"]),
+    ]
+    period_values = [
+        "1",
+        "Break after 1",
+        "2",
+        "Break after 2",
+        "3",
+        "Break after 3",
+        "4",
+        "Break after 4",
+        "5",
+    ]
+    tracking_data["period"] = np.select(period_conditions, period_values)
```

Comment:
This is double code, if it is necesary here this should be a loose function. But maybe it is cleaner to move this to the tracab.py and metrica_tracking_data.py themselves?